HPXMLtoOpenStudio/resources/lighting.rb

```diff
@@ -62,23 +74,36 @@ def self.apply(model, weather, spaces, lighting_groups, usage_multiplier, eri_ve
       ltg.setSpace(garage_space)
       ltg.setEndUseSubcategory(Constants.ObjectNameGarageLighting)
       ltg_def.setName(Constants.ObjectNameGarageLighting)
-      ltg_def.setLightingLevel(space_design_level)
+      ltg_def.setLightingLevel(design_level)
       ltg_def.setFractionRadiant(0.6)
       ltg_def.setFractionVisible(0.2)
       ltg_def.setReturnAirFraction(0.0)
-      ltg.setSchedule(sch.schedule)
+      ltg.setSchedule(garage_sch.schedule)
     end
 
     if ext_kwh > 0
-      space_design_level = sch.calcDesignLevel(sch.maxval * ext_kwh)
+      design_level = exterior_sch.calcDesignLevelFromDailykWh(ext_kwh / 365.0)
 
       # Add exterior lighting
       ltg_def = OpenStudio::Model::ExteriorLightsDefinition.new(model)
       ltg = OpenStudio::Model::ExteriorLights.new(ltg_def)
       ltg.setName(Constants.ObjectNameExteriorLighting)
       ltg_def.setName(Constants.ObjectNameExteriorLighting)
-      ltg_def.setDesignLevel(space_design_level)
-      ltg.setSchedule(sch.schedule)
+      ltg_def.setDesignLevel(design_level)
+      ltg.setSchedule(exterior_sch.schedule)
+    end
+
+    # Add exterior holiday lighting
+    if not lighting.exterior_holiday_daily_energy_use.nil?
+      design_level = exterior_holiday_sch.calcDesignLevelFromDailykWh(lighting.exterior_holiday_daily_energy_use.to_f)
```

Comment:
I feel like `lighting.exterior_holiday_daily_energy_use` should already be a float...